scripts/health.sh

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env bash
+# Check the health status of the current environment
+# If this script executed successfully, ware all set to run.
+set -u
+
+MAJOR_MINIMUM_VERSION_DOCKER="17"
+MINOR_MINIMUM_VERSION_DOCKER="05"
+
+help() {
+  if [ "$1" == "docker" ]; then
+    echo "Please follow the installation instructions on https://docs.docker.com/get-docker/"
+    echo "or for a quick installation for non production linux machines the instruction on https://github.com/docker/docker-install"
+  else
+    echo "There is no suggestion for the issue at hand, please contact your administrator or support"
+  fi
+}
+
+is_command_available () {
+  command -v "$1" > /dev/null
+}
+
+is_version_compatible() {
+  local min_major="$1" # $1 - expected min major version
+  local min_minor="$2" # $2 - expected min minor version
+  local major="$3"     # $3 - actual major version
+  local minor="$4"     # $4 - actual minor version
+  if [ "$major" -lt "$min_major" ]; then
+    return 1
+  fi
+  if [ "$major" -eq "$min_major" ] && [ "$minor" -ge "$min_minor" ]; then
```

Comment:
still looks wrong to me.
If actual minor version is greater or equal than minimum minor version then it should succeed and not exit with 1.

Reply:
yes, it needs to be
```suggestion
  if [ "$major" -eq "$min_major" ] && [ "$minor" -lt "$min_minor" ]; then
```

Reply:
Arrrrg, must haven been sneaked in during the renaming/rework. good catch again! Thx @ the both of you